client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import _orderBy from 'lodash/orderBy';
+import Grid from '@material-ui/core/Grid';
+import { Box } from '@material-ui/core';
+import store from 'store';
+import { Table } from '../../components/generic';
+
+const columns = [
+  { id: 'participantId', name: 'ID' },
+  { id: 'pptName', name: 'Name' },
```

Comment:
Nitpick again: naming here
```suggestion
  { id: 'participanttName', name: 'Name' },
```

Reply:
Yeah i'm also in favour of longer names. It's just too easy to get lost when a new developer joins the project, or when it hits maintenance and you don't look at it for a couple of months